index.html

```diff
@@ -981,9 +981,16 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
-            Untrusted events have their <a>coalesced events list</a> initialized to the value passed to the
-            constructor.</p>
+            that were coalesced into this event. The "parent" trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> event represents an accumulation of these coalesced events,
+            but may have additional processing (for example to align with the display refresh rate).
+            For all other trusted event types, it is an empty list. Untrusted events have their
+            <a>coalesced events list</a> initialized to the value passed to the constructor.</p>
+
+            <div class="note">Since a trusted parent event is a summary or aggregation of the
+            coalesced events, developers should only need to process either the parent events
+            or all of the coalesced events, but not both.
```

Comment:
...this kind of hints that there should be at least one event, since otherwise this comment wouldn't work, but perhaps we should be very explicit about non-empty coalesced events list.